### ISSUE TITLE
Does the training set of pix2cap-coco contain images of refcocog validation set?

### ISSUE THREAD

**Issue by CharlesGong12** [geshang777/pix2cap#1]
Hi, 

Thanks for your amazing work! I'm wondering if the training set portion of your proposed dataset will contain images from the validation set of existing datasets, such as validation set of refcocog.

**Comment by geshang777** [comment]
Thanks for the interest.  Our dataset is constructed exclusively based on the Panoptic COCO 2017, adhering to its original train/validation splits, and does not include any images or captions from additional datasets.
